### PR TITLE
Remove duplicate code in `printing/tests/test_latex.py`

### PR DESCRIPTION
**Pull request by Jatin2020-24**
#### Brief description of what is fixed or changed
Removed re-declaration of symbols and moved symbol declaration to (L 63,L 64)

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by Jatin2020-24** [comment]
@moorepants @sidhu1012 Please review this PR.

**Comment by sidhu1012** [comment]
This line could also be removed .
https://github.com/sympy/sympy/blob/4875c64ff6249da145133f16e6b733a462139bff/sympy/printing/tests/test_latex.py#L2640

**Comment by Jatin2020-24** [comment]
@sidhu1012 Thanks for pointing out duplication.

**Comment by Jatin2020-24** [comment]
@sidhu1012 Please review.

**Comment by sidhu1012** [comment]
LGTM

**Review by sidhu1012 — APPROVED** [review]

**Comment by 0sidharth** [comment]
Although I'm not sure if this change should be a PR, there are still multiple places in `sympy/printing/tests/test_latex.py` having redeclaration of these symbols (L803, L1315, ...).

**Comment by Jatin2020-24** [comment]
> Although I'm not sure if this change should be a PR, there are still multiple places in `sympy/printing/tests/test_latex.py` having redeclaration of these symbols (L803, L1315, ...).

We should declare symbols every time in functions separately so readability of file is  maintained.

**Comment by 0sidharth** [comment]
How does redefining a generic symbol like `x` with no added assumptions help in readability?

**Comment by Jatin2020-24** [comment]
@0sidharth All duplicates removed as suggested by you.

File: sympy/printing/tests/test_latex.py [hunk 1]
```diff
@@ -60,8 +60,8 @@ class lowergamma(sym.lowergamma):
     pass   # testing notation inheritance by a subclass with same name
 
 
-x, y, z, t, w, a, b, c, s, p = symbols('x y z t w a b c s p')
-k, m, n = symbols('k m n', integer=True)
+x, y, z, t, w, a, b, c, s, p, x1, x2, n1, n2, d, k, f, i, j, e = symbols('x y z t w a b c s p x1 x2 n1 n2 d k f i j e')
+k, m, n, f, g, h = symbols('k m n f g h', integer=True)
```
Comment (0sidharth) [inline sympy/printing/tests/test_latex.py:64]: I don't think g and h are being used anywhere apart from `test_latex_decimal_separator`, where they are being redefined anyways. Also, f should not be declared as an integer (L2415), and you are defining it just above anyways.

File: sympy/printing/tests/test_latex.py [hunk 2]
```diff
@@ -60,8 +60,8 @@ class lowergamma(sym.lowergamma):
     pass   # testing notation inheritance by a subclass with same name
 
 
-x, y, z, t, w, a, b, c, s, p = symbols('x y z t w a b c s p')
-k, m, n = symbols('k m n', integer=True)
+x, y, z, t, w, a, b, c, s, p, x1, x2, n1, n2, d, k, f, i, j, e = symbols('x y z t w a b c s p x1 x2 n1 n2 d k f i j e')
```
Comment (0sidharth) [inline sympy/printing/tests/test_latex.py:63]: Ordering it alphabetically might be better, but it doesn't make much of a difference so can be left also.

Comment (Jatin2020-24) [reply sympy/printing/tests/test_latex.py]: Nice suggestion

**Comment by Jatin2020-24** [comment]
@czgdp1807 Please take a look when you have the time.

File: sympy/printing/tests/test_latex.py [hunk 3]
```diff
@@ -1673,11 +1655,7 @@ def test_PrettyPoly():
 
 
 def test_integral_transforms():
-    x = Symbol("x")
-    k = Symbol("k")
```
Comment (smichr) [inline sympy/printing/tests/test_latex.py:1677]: With the changes made, the `k` is now has `integer=True`.

File: sympy/printing/tests/test_latex.py [hunk 4]
```diff
@@ -2220,7 +2197,6 @@ def test_MatrixElement_printing():
     F = C[0, 0].subs(C, A - B)
     assert latex(F) == r"\left(A - B\right)_{0, 0}"
 
-    i, j, k = symbols("i j k")
```
Comment (smichr) [inline sympy/printing/tests/test_latex.py:2223]: these are now all integers -- that is probably ok here

Comment (Jatin2020-24) [reply sympy/printing/tests/test_latex.py:2223]: can this PR be merged

**Comment by JSS95** [comment]
Any more review?

**Comment by oscarbenjamin** [comment]
This seems relevant: https://github.com/sympy/sympy/pull/21211#discussion_r608884234

**Pull request closed without merging**